### PR TITLE
interToneGap and duration fixes

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issues https://github.com/w3c/webrtc-pc/issues/770, https://github.com/w3c/webrtc-pc/issues/772 and https://github.com/w3c/webrtc-pc/issues/773

Need to evaluate PR https://github.com/w3c/webrtc-pc/pull/852 before merging (if merged, can close this PR).

### CODE REVIEW

**Comment by stefhak** [comment]
LGTM.

**Comment by alvestrand** [comment]
As written, the call

insertDtmf('a', duration=1000)
insertDtmf('b')

will result in b being sent out with a duration of 100 (the default).
Users may expect that the attribute is sticky, so that b is sent out with duration 1000.

We need to be clear about the choice.

File: webrtc.html [hunk 1]
```diff
@@ -7729,7 +7729,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       <div>
         <pre class="idl">
 interface RTCDTMFSender : EventTarget {
-    void insertDTMF (DOMString tones, optional unsigned long duration = 100, optional unsigned long interToneGap = 70);
+    void insertDTMF (DOMString tones, optional unsigned long duration, optional unsigned long interToneGap);
```
Comment (alvestrand) [inline webrtc.html]: This needs to be reverted.

File: webrtc.html [hunk 2]
```diff
@@ -7757,24 +7757,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
...
-              attribute MUST return the current tone duration value. This value
-              will be the value last set via the <code><a>insertDTMF</a></code>
-              method, or the default value of 100 ms if
-              <code><a>insertDTMF</a></code> was called without specifying the
-              duration.</p>
+              <p>The <dfn id="dom-RTCDTMFSender-duration">
+              <code>duration</code></dfn> attribute MUST return the current
+              tone duration value in ms. This value will be the value last set
+              via the <code><a>insertDTMF</a></code> method, or the default value
+              of 100 ms if <code><a>insertDTMF</a></code> was not called or
```
Comment (alvestrand) [inline webrtc.html]: Remove all text after the comma.

File: webrtc.html [hunk 3]
```diff
@@ -7757,24 +7757,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
...
               <p>The <dfn id=
               "dom-RTCDTMFSender-intertonegap"><code>interToneGap</code></dfn>
-              attribute MUST return the current value of the between-tone gap.
-              This value will be the value last set via the
+              attribute MUST return the current value of the between-tone gap
+              in ms. This value will be the value last set via the
               <code><a>insertDTMF</a></code> method, or the default value of 70
-              ms if <code><a>insertDTMF</a></code> was called without
-              specifying the interToneGap.</p>
+              ms if <code><a>insertDTMF</a></code> was not called or called
```
Comment (alvestrand) [inline webrtc.html]: Remove all text after the comma.

Comment (aboba) [inline webrtc.html]: Reverted text after the comma to what is currently in the draft.

File: webrtc.html (hunk 2 above)
Comment (aboba) [inline webrtc.html]: Reverted text after the comma to what is currently in the draft.

File: webrtc.html (hunk 1 above)
Comment (aboba) [inline webrtc.html]: Reverted.

**Comment by aboba** [comment]
May not need this PR, since response to Issue https://github.com/w3c/webrtc-pc/issues/843 seems to suggest that we can dispense with the duration and interToneGap attributes.

**Comment by aboba** [comment]
Since we have removed the duration and interToneGap attributes, I think this PR is now moot.

**Pull request closed without merging**